controller/search_blackbox_test.go

```diff
@@ -1045,7 +1044,6 @@ func (s *searchControllerTestSuite) TestSearchQueryScenarioDriven() {
 		_, result := test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
 		require.NotNil(s.T(), result)
 		require.NotEmpty(t, result.Data)
-		assert.Len(t, result.Data, 1)
```

Comment:
Why was this removed?

Reply:
@jarifibrahim because the test query was too broad and this expectation is not always true.